includes/Modules/Site_Verification.php

```diff
@@ -380,44 +386,42 @@ protected function setup_services( Google_Client $client ) {
 	 * Handles receiving a verification token for a user by the authentication proxy.
 	 *
 	 * @since 1.1.0
+	 * @since n.e.x.t Runs on `admin_action_googlesitekit_proxy_setup` and no longer redirects directly.
 	 */
 	private function handle_verification_token() {
-		$authentication = $this->authentication;
-		$auth_client    = $authentication->get_oauth_client();
+		$verification_token = $this->context->input()->filter( INPUT_GET, 'googlesitekit_verification_token' );
+		$verification_nonce = $this->context->input()->filter( INPUT_GET, 'googlesitekit_verification_nonce' );
+		$verification_type  = $this->context->input()->filter( INPUT_GET, 'googlesitekit_verification_token_type' ) ?: self::VERIFICATION_TYPE_META;
```

Comment:
Let's split this last assignment into multiple lines. I'd advise to refrain from using too many ternary operators. It makes the code harder to read and understand especially if the lines are already long, like here. Let's try to be verbose when it would otherwise be "too clever".

We can assign it first, and then just set it to meta if empty.

Reply:
I've updated this to be on separate lines as requested:

```diff
- $verification_type  = $this->context->input()->filter( INPUT_GET, 'googlesitekit_verification_token_type' ) ?: self::VERIFICATION_TYPE_META;
+ $verification_type  = $this->context->input()->filter( INPUT_GET, 'googlesitekit_verification_token_type' );
+ $verification_type  = $verification_type ?: self::VERIFICATION_TYPE_META;
```

> I'd advise to refrain from using too many ternary operators.

Would you please clarify what you mean by this?